### ISSUE TITLE
What exactly does "continual pretrained" mean? 

### ISSUE THREAD

**Issue by krejciadam** [bowang-lab/scGPT#143]
Hi!
  I noticed the new models and zero-shot tutorials. The results look interesting. One thing I can't figure out is what exactly do you mean by "continual pretrained" checkpoint. 

The term evokes the idea of being trained on non-binned (_continuous_) data, but from here: https://github.com/bowang-lab/scGPT/tree/main/tutorials/zero-shot I understand that it is just trained a little more with more lables (the training _continued_). But then the text also mentions that it: `takes types and technical batch effects into consideration intrinsically`
which sounds like maybe batch labels were used during pre-training? I think the pre-training code would answer all my questions, but I can't find it. Is that available somewhere? Thank you!

**Comment by subercui** [comment]
Hi, thanks for the question and your interest in this new extension. We kept a relatively brief description in the tutorial. The new continual pretraining merges the pretraining workflow with the additional cell type classification, as you said. The text you highlighted intends to mean that the model can learn to better understand cell types across multiple technical batches since the target cell type labels are annotated and curated in such a way across batches. Therefore, in this way, the model can learn this intrinsically. That's also what we showed in the tutorial through the comparison between the continual pretrained model and the original one.  

Since this is a new extension, it is not included in the pre-training code yet.

I think we can definitely release the new code for this. @Kuan-Pang Please comment on this and do you think it is possible since you created most of this
